Makefile

```diff
@@ -504,10 +504,13 @@ promtail-push: promtail-image-cross
 	$(call push-image,promtail)
 
 # loki
+BUILD_PLATFORM ?= amd64
```

Comment:
I think maybe this should be 'TARGET_PLATFORM' as it determines what the architecture the age will run on (not what it's built on)

Reply:
Agree 👍  Update in https://github.com/grafana/loki/pull/4981/commits/91b74a5f08c452f7ae561cd6b9288eb63e47a3dd